docs/getting-started/quickstart/README.md

```diff
@@ -0,0 +1,211 @@
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/dot-net/dot-net-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-.Net?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab node.js
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/nodejs/nodejs-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Node.js?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+:::
+
+2. Follow the instructions to create a new Codespace.
+
+3. Wait for your Codespace to build. This will take a few moments and you will see a welcome message in the terminal.
+
+## Step 2: Start the EventStoreDB Server
+
+1. Once your Codespace is loaded, run the following custom script in the terminal to start the EventStoreDB server:
+
+   ```sh
+   ./start-cluster.sh
+   ```
+
+   This script will: 
+   - Pull the EventStoreDB server Docker container,
+   - Start the server, and
+   - Print the URL to the EventStoreDB Admin UI. 
+
+2. Open your browser and navigate to the printed URL. This will display the EventStoreDB Admin UI.
+3. Keep the Admin UI open for the next steps.
+4. (Optional) Review the content of custom script.
+
+## Step 3: Install Required Pacakge for Sample Code
+
+1. Run this command to install the package for EventStoreDB client. This will be used in the sample codes:
+
+::: tabs#dev-language-platform
+@tab Select >
+
+@tab Python
+   ```sh
+   pip install -r requirements.txt
+   ```
+
+@tab Java
+   ```sh
+   mvn package
+   ```
+
+@tab .NET
+   Step not required
+   
+@tab node.js
+   ```sh
+   yarn install
+   ```
+:::
+
+## Step 4: Append an Event to EventStoreDB
+
+1. Run this command to execute the sample. This appends an event to EventStoreDB:
+
+::: tabs#dev-language-platform
+@tab Select >
+
+@tab Python
+   ```sh
+   python3 sample_append.py
+   ```
+
+@tab Java
+   ```sh
+   java -cp \
+      target/eventstoredb-demo-1.0-SNAPSHOT-jar-with-dependencies.jar \
+      com.eventstoredb_demo.SampleWrite
+   ```
+
+@tab .NET
+   ```sh
+   dotnet run --project SampleAppend/
+   ```
+
+@tab node.js
+   ```sh
+   node sample_append.js
+   ```
+:::
+
+2. You should see a confirmation for the event append.
+3. (Optional) Review the content of sample code.
+
+## Step 5: Verify the Appended Event in the Admin UI
+
+1. Return to the Admin UI and click the **Stream Browser** link from the top navigation bar.
```

Comment:
Screenshot

Reply:
I've decided not to do this one to avoid adding too many screen shots to the guide. In general, I prefer not to have too many screenshots because they can be hard to version and maintain.